htlcswitch/test_utils.go

```diff
@@ -2,7 +2,7 @@ package htlcswitch
 
 import (
 	"bytes"
-	"crypto/rand"
+	crand "crypto/rand"
```

Comment:
New standard for naming crypto rand?

Reply:
the linter kept mistaking it for `match/rand`

Reply:
They should totally have distinct packages name imo...